helperMethods.txt

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
nit: maybe `.tmpl` instead of `.txt`?  `.txt` is something that normally contains documentation, users may be confused.

Reply:
sure, thanks!

Reply:
I like `.ts.tmpl` to show the type of template, and the fact that its a template